distribution/Makefile

```diff
@@ -36,6 +36,10 @@ docker-cloud-container-check: containers container-check-prereqs
 	ENVIRONMENT=docker-cloud ./tests/offline-tests.sh
 	ENVIRONMENT=docker-cloud ./tests/tests.sh
 
+aws-cloud-container-check: containers container-check-prereqs
+	ENVIRONMENT=aws-ec2-cloud ./tests/offline-tests.sh
+	ENVIRONMENT=aws-ec2-cloud ./tests/tests.sh
```

Comment:
Not sure what exactly these do, but I would expect something to fail if

* there is a mismatch in plugin dependencies within this environment
* the JEP-201 YAML has a typo

Reply:
> there is a mismatch in plugin dependencies within this environment

Tried to test this, but was unable to: `ingest-update-center` seems to be completely ignoring the versions actually specified in `essentials.yaml`, unless I am missing something.

Reply:
> the JEP-201 YAML has a typo

Tried to test this. The build did fail, though with no coherent error message that I could identify.

```
…
Waiting for Jenkins startup a bit more... (38/40 attempts) 
Waiting for Jenkins startup a bit more... (39/40 attempts) 
Waiting for Jenkins startup a bit more... (40/40 attempts) 
**** Jenkins did not start before timeout. Tests are expected to fail. ****
test_smoke
test_required_plugins_are_here
/evergreen/jenkins/home/plugins/metrics.hpi
/evergreen/jenkins/home/plugins/essentials-0.3-rc20.a3f76cdd1348.hpi
/evergreen/jenkins/home/plugins/configuration-as-code-0.11-alpha-rc362.942711740b07.hpi
test_no_node_error_in_logs
test_no_executor
test_plugins_are_not_exploded_under_jenkins_home
test_war_is_not_exploded_under_jenkins_home
test_logs_are_not_under_jenkins_home
test_jenkins_logs_is_found_on_disk
ASSERT:expected:<0> but was:<1>
shunit2:ERROR test_jenkins_logs_is_found_on_disk() returned non-zero return code.
test_essentials_telemetry_logging_is_found_on_disk
test_login_http_200
ASSERT:expected:<200> but was:<500>
shunit2:ERROR test_login_http_200() returned non-zero return code.
test_instance_identity_http_200
ASSERT:expected:<200> but was:<500>
shunit2:ERROR test_instance_identity_http_200() returned non-zero return code.
test_metrics_health_check
ASSERT:expected:<200> but was:<500>
Error: Parse error on line 1:
    <!DOCTYPE html><html
----^
Expecting 'STRING', 'NUMBER', 'NULL', 'TRUE', 'FALSE', '{', '[', got 'undefined'
    at Object.parseError (/usr/lib/node_modules/jsonlint/lib/jsonlint.js:55:11)
    at Object.parse (/usr/lib/node_modules/jsonlint/lib/jsonlint.js:132:22)
    at parse (/usr/lib/node_modules/jsonlint/lib/cli.js:82:14)
    at Socket.<anonymous> (/usr/lib/node_modules/jsonlint/lib/cli.js:149:41)
    at emitNone (events.js:111:20)
    at Socket.emit (events.js:208:7)
    at endReadableNT (_stream_readable.js:1056:12)
    at _combinedTickCallback (internal/process/next_tick.js:138:11)
    at process._tickCallback (internal/process/next_tick.js:180:9)
ASSERT:expected:<0> but was:<1>
parse error: Invalid numeric literal at line 7, column 12
ASSERT:expected:<true> but was:<>
shunit2:ERROR test_metrics_health_check() returned non-zero return code.
test_logs_are_propagated
test_evergreen_home_is_fully_owned_by_jenkins_user
test_error_telemetry_service_is_secured
test_docker_available_as_jenkins_user
Cannot connect to the Docker daemon at tcp://localhost:2375. Is the docker daemon running?
/usr/local/bin/docker: Cannot connect to the Docker daemon at tcp://localhost:2375. Is the docker daemon running?.
See '/usr/local/bin/docker run --help'.

Retrieving test run data (container logs, ...) before shutdown... Will be in the build/tests-run-2018-07-17_12-05-02-aws-ec2-cloud-tests.sh directory... Done getting logs for distribution_backend_1
distribution_db_1
distribution_instance_1
Stopping distribution_instance_1 ... done
Stopping distribution_backend_1  ... done
Stopping distribution_db_1       ... done
Removing distribution_instance_1 ... done
Removing distribution_backend_1  ... done
Removing distribution_db_1       ... done
Removing network distribution_default
G'day!

Ran 16 tests.
./shunit2/shunit2: line 950: [: skipped=5: integer expression expected

FAILED (failures=10)
Makefile:40: recipe for target 'aws-cloud-container-check' failed
make: *** [aws-cloud-container-check] Error 1
```